lib/webdrivers/chromedriver.rb

```diff
@@ -44,7 +41,7 @@ def latest_point_release(version)
           msg = version > latest_release ? 'you appear to be using a non-production version of Chrome; ' : ''
           msg = "#{msg}please set `Webdrivers::Chromedriver.version = <desired driver version>` to an known "\
```

Comment:
Needs to be update if swapping to `required_version`

Reply:
I was going to address everything from #36 is a separate PR